tests/python/relay/test_pass_partition_graph.py

```diff
@@ -862,7 +863,7 @@ def expected():
     test_extern_ccompiler_default_ops()
     test_extern_ccompiler()
     test_extern_dnnl()
-    test_extern_dnnl_mobilenet()
+    #test_extern_dnnl_mobilenet()
```

Comment:
What change in this PR makes running mobilenet test difficult?

Reply:
Ah I missed one line in this separated PR. We added `mod["main"] = relay.build_module.bind_params_by_name(mod["main"], params)` to this case and it results in the large contant tensor issue that we didn't handle it in this PR.